layouts/partials/cards/encounters/fight_card.html

```diff
@@ -0,0 +1,25 @@
+<div class="relative mb-8 overflow-hidden">
+  {{ if or (eq .Params.coming_soon true) (eq .Params.spoiler true)}}
+    <a class="block bg-card-light">
+  {{ else }}
+    <a class="block bg-card-light" href="{{ .Permalink }}">
+  {{ end }}
```

Comment:
I don't think we want to remove the `href` if it's just a spoiler — only if it's coming soon

Also I feel like this is a super dumb nit from me, but:

```html
<a class="block bg-card-light"> {{ if (eq .Params.coming_soon false) }}href="{{ .Permalink }}"{{ end }}>
```

Reply:
My editor is colouring it weirdly but it works 😛